confd/production.tmpl

```diff
@@ -1,9 +1,9 @@
 {
     "kafka" : {
         "host_ip": "{{ getv "/kafka/host/ip" "10.28.11.49:9092" }}",
-        "topic": "{{ getv "/kafka/topic" "topic-test-132" }}",
+        "topic": "{{ getv "/kafka/topic" "topic-test-888" }}",
         "group_id": "{{ getv "/kafka/group/id" "task_group_1" }}",
-        "auto_commit": "{{ getv "/kafka/auto/commit" "False" }}",
+        "auto_commit": {{ getv "/kafka/auto/commit" "false" }},
```

Comment:
is there a reason for this param (auto_commit) to be config?

Reply:
i guess not, seems like its should be always False.